libvmi/driver/driver_interface.h

```diff
@@ -138,6 +138,26 @@ typedef struct driver_interface {
     status_t (*set_debug_event_ptr)(
         vmi_instance_t,
         bool enabled);
+    status_t ( *slat_get_domain_state_ptr )(
```

Comment:
Style doesn't match the others here (ie. no spaces before and after the function name)